### ISSUE TITLE
rising fastball ... or not

### ISSUE THREAD

**Issue by JohnDenker** [ericmazur/PnPbook#126]
In section 18.4 on page 481 question 3 asks:

> For a thrown ball to rise, what type of spin should the thrower place on it?

I assume we are talking about baseballs (as opposed to, say Wiffle balls).

There are three problems here:  The question is based on a myth, and the answer is wrong twice over.
- AFAICT there is no reason to believe that a human being can throw a baseball horizontally with enough spin and/or enough airspeed to overcome the force of gravity. 
  
  (FWIW if you want the ball to rise, the obvious solution is to throw it underhand, so that the aim point is higher than the release point.  This has got nothing to do with spin.)
- Farther down on page 481, the answer says that backspin `causes the speed of the air relative to the surface of the ball to be greater at the top`.  This is geometrically impossible.  It's preposterous.
- The answer goes on to say `Consequently the air pressure is lower`.  That is completely wrong physics.  The speed of the air **_relative to the surface**_ has got precisely nothing to do with the pressure exerted on the surface.
  
  (FWIW the relative speed _at the surface_ is zero anyway.  We can invoke the **_no-slip boundary condition**_ which is well known to be valid to tremendous accuracy.  As Feynman pointed out, how do you explain the presence of dust on fan blades?)
  
  The true physics of lift-production has been well understood for over 100 years.   See e.g. https://www.av8n.com/how/htm/airfoils.html

To summarize, this starts with a preposterous observation and then applies a bogus principle to arrive at a mythical conclusion.
